database/insertData/laos/03_template_license_MMC_MMD.js

```diff
@@ -1072,7 +1160,6 @@ exports.queries = [
                         elementTypePluginCode: "shortText"
                         category: QUESTION
                         parameters: { label: "Owner email" }
-                        isEditable: false
                         defaultValue: {
                           operator: "buildObject",
                           properties: [
```

Comment:
Same thing about re-uploading documents (if we allow changing this fields from company/ownership):

### Owner
- **LMMD03** – Medical Certificate (max 3 months)
- **LMMD04** – Certificate of current residence with photo (max 3 months)
- **LMMD05** – Recent (max 1 year) photo
- **LMMD07** – Criminal record "number 3"

### Company
- **LMMD10** – Map of location of company
- **LMMD11** – Layout of facilities
- **LMMD13** – Proof of ownership of facilities or contract of rent
- **LMMD14** – Copy of business licence issued by Ministry of Commerce